Terminal.Gui/Core/Border.cs

```diff
@@ -651,39 +692,51 @@ private void DrawChildBorder (Rect frame, bool fill = true)
 
 			// Draw the upper BorderThickness
 			for (int r = frame.Y - drawMarginFrame - sumThickness.Top;
-				r > 0 && r < frame.Y - drawMarginFrame - padding.Top; r++) {
+				r < frame.Y - drawMarginFrame - padding.Top; r++) {
+
+				if (r < 0) {
+					continue;
+				}
 				for (int c = frame.X - drawMarginFrame - sumThickness.Left;
-					c > 0 && c < Math.Min (frame.Right + drawMarginFrame + sumThickness.Right, driver.Cols); c++) {
+					c < Math.Min (frame.Right + drawMarginFrame + sumThickness.Right, driver.Cols); c++) {
```

Comment:
The problem it's was still crashing because it call the `AddRuneAt (driver, c, r, ' ');` where 'r' values was negative. It was throwing after I changed to `Y = Pos.Center` and resizing the terminal. Maybe is needed both?

Reply:
@tig I added your previous code with my changes and it isn't working well. See the gif. The reason is that your code doesn't runs the valid rows afterwards. That why we have to use `continue` instruction.

![border-issue](https://user-images.githubusercontent.com/13117724/222503175-20dd9e5c-15c3-4d30-abdd-768f6a2406c0.gif)

Reply:
Also fixes this other issue. Until all the lines and columns are processed, they can't bypass the `for` statement, but only `continue` until all them are processed, ignoring the negative values.

![border-left-issue](https://user-images.githubusercontent.com/13117724/222549736-b51d2c17-da86-4341-a849-db133a467b78.gif)